src/game_interpreter_map.cpp

```diff
@@ -338,7 +339,8 @@ bool Game_Interpreter_Map::CommandEndShop(RPG::EventCommand const& com) { // cod
 bool Game_Interpreter_Map::CommandShowInn(RPG::EventCommand const& com) { // code 10730
 	int inn_type = com.parameters[0];
 	Game_Temp::inn_price = com.parameters[1];
-	Game_Temp::inn_handlers = com.parameters[2] != 0;
+	// Not used, but left here for documentation purposes
+	bool has_inn_handlers = com.parameters[2] != 0;
```

Comment:
This could be commented out as in `has_shop_handlers` for consistency.

Reply:
Fixed